virtual_rainforest/data_variables.toml

```diff
@@ -304,23 +335,23 @@ used_by = [ "abiotic", "soil"]
 
```

Comment:
Don't know what we want to do with this `soil_respiration` variable. It isn't currently produced by the `soil` model (though it could be fairly straightforwardly), but I guess it makes sense to keep it as it is something that the full `abiotic` model would want eventually?

Reply:
A lot of the variables are stretch goals, including all the respiration. I kept them despite knowing that we will not use them in the near future.